.eslintrc.js

```diff
@@ -115,7 +117,9 @@ module.exports = {
         '@typescript-eslint/explicit-function-return-type': 'off',
         '@typescript-eslint/naming-convention': 'off',
         '@typescript-eslint/no-unsafe-call': 'off',
-        '@typescript-eslint/no-floating-promises': 'off'
+        '@typescript-eslint/no-floating-promises': 'off',
+        'indent': 'off',
+        'array-bracket-spacing': 'off',
```

Comment:
This rule was deprecated in ESLint v8.53.0 and became as default in https://eslint.style/packages/default

Reply:
and so why disabling them?

Reply:
It's for e2e tests, otherwise need to fix about 40 errors. Do you want to do in this PR?